experimental/dds/tree2/api-report/tree2.api.md

```diff
@@ -1705,21 +1680,9 @@ export class SharedTreeFactory implements IChannelFactory {
     readonly type: string;
 }
 
-// @alpha
-export interface SharedTreeMap<TSchema extends MapNodeSchema> extends ReadonlyMap<string, ProxyField<TSchema["info"], "sharedTree", "notEmpty">> {
-    delete(key: string): void;
-    set(key: string, value: ProxyField<TSchema["info"], "sharedTree", "notEmpty"> | undefined): void;
-}
-
-// @alpha
-export type SharedTreeNode = TreeList<AllowedTypes> | SharedTreeObject<ObjectNodeSchema> | SharedTreeMap<MapNodeSchema>;
-
-// @alpha
-export type SharedTreeObject<TSchema extends ObjectNodeSchema, API extends "javaScript" | "sharedTree" = "sharedTree"> = ObjectFields<TSchema["objectNodeFieldsObject"], API>;
-
 // @alpha
 export interface SharedTreeObjectFactory<TSchema extends TreeNodeSchemaBase> {
```

Comment:
This would be a good time to rename this to `TreeObjectNodeFactory`

Reply:
I'll get that in another PR